netci.groovy

```diff
@@ -3,15 +3,26 @@ import jobs.generation.Utilities
 def project = GithubProject
 def branch = GithubBranchName
 def isPr = true
-def containerOs = 'Debian'
-def newJobName = Utilities.getFullJobName(project, containerOs, isPr)
+def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:WindowsServerCore']
 
-def newJob = job(newJobName) {
-    steps {
-        shell("./build-and-test.sh")
-    }
-}
+platformList.each { platform ->
+	def(hostOS, containerOS) = platform.tokenize(':')
+	def newJobName = Utilities.getFullJobName(project, containerOS, isPr)
+
+	def newJob = job(newJobName) {
+		steps {
+			if (hostOS == 'Windows_2016')
+			{
+				shell("powershell -NoProfile -Command .\\build-and-test.ps1")
```

Comment:
Yeah don't use shell for that.  shell actually means *Nix shell.


You can either use batchFile("powershell blah blah") or powershell("blah blah")